### PR TITLE
github: Provide more suggestions for info to include for a test failure

### PR DESCRIPTION
**Pull request by aixtools**
##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

**Review by sxa — COMMENTED** [review]
I'm ok with this, but I'd like @smlambert's approval on the extras before merging so I'll hold off giving my own formal approval for now in case she's not keen on any of the wording etc. :-)

**Review by sxa — COMMENTED** [review]
I'm ok with this, but I'd like @smlambert's approval on the extras before merging so I'll hold off giving my own formal approval for now in case she's not keen on any of the wording etc. :-)

File: .github/ISSUE_TEMPLATE/testcasefail.md
Comment (karianna) [inline .github/ISSUE_TEMPLATE/testcasefail.md:14]: This isn't always the case (only functional tests would have this link) and they expire. Recommend removing this.

File: .github/ISSUE_TEMPLATE/testcasefail.md
Comment (karianna) [inline .github/ISSUE_TEMPLATE/testcasefail.md:15]:
```suggestion
- **Hyperlink** to the Grinder re-run of the test with the failure, if available.
```

**Review by karianna — CHANGES_REQUESTED** [review]
Nitpicks :-)

File: .github/ISSUE_TEMPLATE/testcasefail.md
Comment (aixtools) [reply .github/ISSUE_TEMPLATE/testcasefail.md:14]: * I am trying to address the specific issues I had when revisiting this issue: https://github.com/adoptium/infrastructure/issues/2153. 
* I am struggling with how to specify that the link should not be: https://ci.adoptopenjdk.net/view/Test_functional/job/Test_openjdk8_hs_special.functional_ppc64_aix/28/ (which, as you say expires), but: https://ci.adoptopenjdk.net/view/Test_functional/job/Test_openjdk8_hs_special.functional_ppc64_aix/ which lists the recent runs of the same test.
* PLEASE! help me out here - all the different ways to specify things makes my head spin (and I am trying to make it easier for all of to re-visit testcases).

Comment (aixtools) [reply .github/ISSUE_TEMPLATE/testcasefail.md:14]: * Actually, now I see I only need to strip off (from the example above) the /28/ and the link shows the recent examples - at least for those with access.
* wish I had tried that MUCH earlier - starting at https://ci.adoptopenjdk.net and using the navigation, for the novice I am was cumbersome.
* So, again - I would like to keep this suggestion - but perhaps with an improved help text/hint.

**Comment by aixtools** [comment]
> Nitpicks :-)

please comment in the review section. Thx in advance!

File: .github/ISSUE_TEMPLATE/testcasefail.md
Comment (karianna) [reply .github/ISSUE_TEMPLATE/testcasefail.md:14]: @smlambert I think you had a better path suggestion here perhaps?

**Review by karianna — APPROVED** [review]

**Comment by sxa** [comment]
@smlambert Any further comments on this one?

File: .github/ISSUE_TEMPLATE/testcasefail.md
Comment (smlambert) [reply .github/ISSUE_TEMPLATE/testcasefail.md:14]: 2 problems of including a Jenkins link in an issue reported: 
- links with specific build IDs are transient (we typically only keep the last 5 runs)
- functional is only one type of test we run (we have openjdk, perf, system, external as well on this server), so if we do suggest including a link, we might be better to say something like `a link into a job on https://ci.adoptopenjdk.net showing the failure, such as xxxxx`

Strongly prefer to include the Rerun in Grinder link from the Jenkins job into an issue as that remains 'timeless' versus transient.  And strongly recommend that the link be hyperlinked, since it is painfully long as it includes ALL parameters and their values that were setup and used in the original test job.  One could also use the Rerun in Grinder on same machine link, which fills in the LABEL with the hostname of the machine that hosted the test run when it failed.

rerunLinksToInclude

Upcoming work will refine the "Rerun in Grinder link to also set the TARGET equal to the failed targets, as a greater convenience to triagers, see related: https://github.com/adoptium/aqa-tests/issues/3219

Test triagers are advised to right-click and copy that link and paste it into the issue as a hyperlink.  They are also asked to grab any relevant or interesting console output from the failing job (this is helpful for intermittent test failures that are difficult to reproduce).   The xxxx_test_output_yyyyy.tar.gz file archived by the test job (shown in above pic) will also contain core files in the case of a crash, which may in some cases be useful to download and look at when debugging.

**Review by smlambert — COMMENTED** [review]
Added some (hopefully helpful) suggestions

**Comment by aixtools** [comment]
I'll try to get back to this next week, and see if I can work in @smlambert comments/suggestions while work continues on the grinder interface (and this would need updating again).

A couple of key points is to mention that straight links are transient (which is why we want 'better'), and perhaps stress the term 'triage' and what is needed for 'triage' starting from two days to two months after the issue is initially posted.

Other `keywords` I am overlooking?

File: .github/ISSUE_TEMPLATE/testcasefail.md
Comment (sxa) [inline .github/ISSUE_TEMPLATE/testcasefail.md:15]: As per Shelley's comment:

```suggestion
- **Hyperlink** to re-run in Grinder: 
```

File: .github/ISSUE_TEMPLATE/testcasefail.md
Comment (sxa) [inline .github/ISSUE_TEMPLATE/testcasefail.md:14]:
```suggestion
- a link into recent `Test_` job on https://ci.adoptopenjdk.net which showed the failure
```

**Review by sxa — CHANGES_REQUESTED** [review]
Two explcit suggestions based on others' review comments. Is there anything else you want to add to this PR @aixtools ?

**Review by sxa — APPROVED** [review]
Making an executive decision to push through these changes as they are a good addition. We can improve them later via another PR if desired :-)

**Pull request merged**
